library/hatomap.py

```diff
@@ -0,0 +1,413 @@
+from __future__ import annotations
+
+import math
+import random
+import string
+from dataclasses import dataclass, field
+from multiprocessing import Pool
+
+from typing import Any, List, Optional, Tuple, Union
+
+from cv2 import cv2
```

Comment:
```suggestion
import cv2
```

ここは元に戻すのがよさそう  (メモ程度に)。